swift/Sources/FlatBuffers/FlatBufferBuilder.swift

```diff
@@ -50,7 +50,7 @@ public struct FlatBufferBuilder {
   private var serializeDefaults: Bool
 
   /// Current alignment for the buffer
-  var _minAlignment: Int = 0 {
+  @usableFromInline var _minAlignment: Int = 0 {
     didSet {
       _bb.alignment = _minAlignment
     }
```

Comment:
I dont get how this would change the way the compiler acts other than exposing the variable:

```
Like the public access-level modifier, this attribute exposes the declaration as part of the module’s public interface. Unlike public, the compiler doesn’t allow declarations marked with usableFromInline to be referenced by name in code outside the module, even though the declaration’s symbol is exported. However, code outside the module might still be able to interact with the declaration’s symbol by using runtime behavior.
```

- Can you please clarify more?
- Can you provide an example project where is actually happens and breaks the compiler or the code?